docs/StardustDocs/topics/collectionsInterop.md

```diff
@@ -87,6 +95,6 @@ val result = df2.toListOf<Output>()
 
 <!---END-->
 
-### Converting columns with objects instances to ColumnGroup
```

Comment:
I can't reply there, but at line 80, you're missing a "the" before dataframe plugin